termination/gceTerminationHandler.go

```diff
@@ -133,11 +134,12 @@ func (g *gceTerminationSource) handleMaintenanceEvents(state string, exists bool
 		return nil
 	}
 	glog.Infof("Handling maintenance event with state: %q", state)
-	if (g.state.NeedsReboot && state != maintenanceEventNone) ||
-		(!g.state.NeedsReboot && state == trueString) {
+	if (g.state.NeedsReboot && (state != maintenanceEventNone && state != maintenanceEventFalse)) || (!g.state.NeedsReboot && state == trueString) {
```

Comment:
The parenthesis before `state != maintenanceEventNone` is not needed.

Reply:
It's just for readability.